main.py

```diff
@@ -1,8 +1,13 @@
 from program import create_playlist, read_playlist, add_music, add_podcast, search_on_playlist, delete_item, \
     delete_playlist, copy_playlist
+# this is too many methods on the same file. you can break it by functionality to different files
 
 while True:
-
+# nice work! short and to the point
+# if you want to add play... what will it do?
+# i suggest you add it . it will go over the songs and for each song it will
+# activate a method play(). that will just print now the song name, and wait a few seconds, then play the next song
+# this method should be on playlist too.

```

Comment:
hey daniel. please take a look at my comments 
its not mandatory for the project grade
its for you :)